Sources/SafeDICore/Generators/ScopeGenerator.swift

```diff
@@ -351,58 +351,43 @@ actor ScopeGenerator: CustomStringConvertible, Sendable {
     private var generateCodeTask: Task<String, Error>?
 
     private var orderedPropertiesToGenerate: [ScopeGenerator] {
-        get throws {
```

Comment:
we no longer need to `throw` here since we do this checking much earlier now. best reviewed with whitespace turned off